cjworkbench/i18n/extract.py

```diff
@@ -95,3 +93,159 @@ def extract_django(fileobj, keywords, comment_tags, options):
                     message_context = None
                 else:
                     yield lineno, None, smart_text(g), comments
+
+
+def extract_python(fileobj, keywords, comment_tags, options):
+    """Extract messages from Python source code.
+    It returns an iterator yielding tuples in the following form ``(lineno,
+    funcname, message, comments)``.
+    
+    
+    Adapted from the corresponding pybabel built-in function
```

Comment:
In the docstring, could you please list every difference between this function and the pybabel built-in function?